### PR TITLE
DOC-11259: Improves content - 'Whats new in 3.1.0

### PR DESCRIPTION
**Pull request by franrob-projects**
- improves content as per feedback from PM
- ticket [DOC-11259](https://issues.couchbase.com/browse/DOC-11259)
- supersedes https://github.com/couchbase/docs-sync-gateway/pull/734

### CODE REVIEW

File: modules/ROOT/pages/whatsnew.adoc [hunk 1]
```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
...
+This release won't cause any issues with existing apps, as it's compatible with older versions. 
+If you have an app that uses bucket-based APIs, you can still upgrade to 3.1, but please note that this API is now deprecated.
 
-== Overview of {sgw-t} 3.1.0
+* *Improved Data Organization and Access Control for Scopes and Collections* +
+Couchbase Mobile now offers Scopes and Collections, allowing more efficient and scalable data organisation within a bucket. 
+This also introduces an improved method of defining and enforcing data xref:access-control-concepts.adoc[Access Control] more granularly. 
+Multi-tenant apps will also experience better scalability and independent data lifecycle management.
 
-// tag::scopes-and-collections[]
```
Comment (osfameron) [inline modules/ROOT/pages/whatsnew.adoc:26]: is this tag included from anywhere else?

File: modules/ROOT/pages/whatsnew.adoc [hunk 2]
```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
...
-  
-In 3.1, Sync Gateway has significantly improved by incorporating *Scopes and Collections* support:
+* *Improved metadata isolation for Scopes and Collections* +
+Sync Gateway 3.1.0 has improved metadata isolation. 
+The system data maintained by Sync Gateway is now stored in the `_default Scope/Collection`, while both the `_default` and user-defined Scope/Collection can be used for application data.
```
Comment (osfameron) [inline modules/ROOT/pages/whatsnew.adoc]: just `_default` should be formatted as code, not "Scope/Collection"

File: modules/ROOT/pages/whatsnew.adoc [hunk 3]
```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
...
+This also introduces an improved method of defining and enforcing data xref:access-control-concepts.adoc[Access Control] more granularly. 
+Multi-tenant apps will also experience better scalability and independent data lifecycle management.
 
-// tag::scopes-and-collections[]
-  
-In 3.1, Sync Gateway has significantly improved by incorporating *Scopes and Collections* support:
+* *Improved metadata isolation for Scopes and Collections* +
+Sync Gateway 3.1.0 has improved metadata isolation. 
+The system data maintained by Sync Gateway is now stored in the `_default Scope/Collection`, while both the `_default` and user-defined Scope/Collection can be used for application data. 
+For more information on the benefits and examples of use cases, click https://www.couchbase.com/blog/scopes-collections-couchbase-mobile/[here].
```
Comment (osfameron) [inline modules/ROOT/pages/whatsnew.adoc]: We don't usually link to "here" I think?
Check on style-and-tone, but I'd recommend the link being more descriptive, like "information on the benefits and examples of use-cases" instead.

File: modules/ROOT/pages/whatsnew.adoc [hunk 4]
```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
...
 
-1. Scopes and Collections Support: Adding Scopes and Collections support allows for an improved method of defining and enforcing data access control. 
-This enhancement ensures that only authorized users can use the appropriate data.
+* *Collection-Level Sync Functions and Scoped User Associations* +
+The xref:sync-function.adoc[Sync Functions] now work on a Collection level, and additional optional fields have been added to the database configurations to support this update. 
+Each database can only have one Scope, which lets users be associated with a Scope and shared across multiple Collections.
```
Comment (osfameron) [inline modules/ROOT/pages/whatsnew.adoc]: This is a little unclear

File: modules/ROOT/pages/whatsnew.adoc [hunk 5]
```diff
@@ -20,57 +20,29 @@ CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
 If you are upgrading from 2.x, please refer to the {upgrading--xref} page.
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+* *New support for Scopes and Collections* +
```
Comment (osfameron) [inline modules/ROOT/pages/whatsnew.adoc:24]: This should perhaps be a definition list?

`New support for Scopes and Collections::
   Couchbase has improved...`

Or a subheading.

Comment (osfameron) [reply modules/ROOT/pages/whatsnew.adoc:24]: Hmm, I see those were there before, not urgent.

File: modules/ROOT/pages/whatsnew.adoc (hunk 1 above)
Comment (osfameron) [reply modules/ROOT/pages/whatsnew.adoc:26]: `modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-0-release-note.adoc:include::ROOT:whatsnew.adoc[tag=scopes-and-collections]`

Comment (franrob-projects) [reply modules/ROOT/pages/whatsnew.adoc:26]: oddly this isn't showing in my editor 😕

Comment (osfameron) [reply modules/ROOT/pages/whatsnew.adoc:26]: The diff (`-` sign and red background) suggests that it's been deleted in this commit.

**Review by osfameron — APPROVED** [review]
GO GO GO

**Pull request merged**